components/brave_wallet/browser/solana_instruction_builder.h

```diff
@@ -47,6 +48,16 @@ std::optional<SolanaInstruction> CreateAssociatedTokenAccount(
 
 }  // namespace spl_associated_token_account_program
 
+namespace bubblegum_program {
+
+std::optional<SolanaInstruction> Transfer(
```

Comment:
I believe I used https://github.com/metaplex-foundation/mpl-bubblegum/blob/5b3cdfc6b236773be70dc1f0b0cb84badf881248/clients/js-solita/src/generated/instructions/transfer.ts#L81, but has to dig through the js a bit understand exactly what was happening since it's abstracted a bit by the library and the serialize function.  Did you want to include this in the source?

Reply:
It's often much easier to understand what is going on when there is some link to reference implementation or docs

Reply:
@nvonpentz please put the link here, it's better than nothing.

Reply:
https://github.com/metaplex-foundation/mpl-bubblegum/blob/ec267f620e634073acaef85a2d41bcc4f5a039ef/programs/bubblegum/program/src/processor/transfer.rs#L32 could be another candidate.